Sources/SocksCore/SocketOptions+Deprecated.swift

```diff
@@ -0,0 +1,100 @@
+//
```

Comment:
Oh sorry, forgot this repo was donated to us. All of our others don't include headers. You can leave it.